mkosi

```diff
@@ -193,7 +193,7 @@ def gpt_root_native(arch: str) -> Tuple[uuid.UUID, uuid.UUID]:
     elif arch == 'aarch64':
         return GPTRootTypePair(GPT_ROOT_ARM_64, GPT_ROOT_ARM_64_VERITY)
     else:
```

Comment:
hmm, the python version check at the top of the script should be updated to 3.6 now, right?